src/pks/transport/transport_ats_pk.cc

```diff
@@ -202,6 +203,17 @@ void Transport_ATS::Setup(const Teuchos::Ptr<State>& S)
     S->RequireField(mass_src_key_, mass_src_key_)->SetMesh(mesh_)->SetGhosted(true)
       ->AddComponent("cell", AmanziMesh::CELL, 1);
     S->RequireFieldEvaluator(mass_src_key_);
+    
+    S->RequireField(geochem_src_factor_key_, geochem_src_factor_key_)->SetMesh(mesh_)->SetGhosted(true)
+      ->AddComponent("cell", AmanziMesh::CELL, 1);
+
+    Teuchos::ParameterList& wc_eval = S->FEList().sublist(geochem_src_factor_key_);
```

Comment:
Can you make this:

    Teuchos::ParameterList& wc_eval = S->GetEvaluatorList(geochem_src_factor_key_);

instead of calling sublist yourself.  This helps us be more general for subgrid models, where resolving this list is a bit weirder (see State.cc:1492, it is a little more complicated).

Reply:
Fixed